src/main/resources/openapi/audioRequest.yaml

```diff
@@ -0,0 +1,60 @@
+openapi: 3.0.1
+
+servers:
+  -  url: http://localhost:4550/
+
+info:
+  description: Modernised DARTS (Digital Audio Recording and Transcription Service).
+  version: "0.0.1"
+  title: Modernised DARTS
+
+paths:
+  /audio-request/addAudioRequest:
```

Comment:
Wondering if the path could be more similar to our Java package naming convention.
So would POST `/audio/request` be better?
I'm suggesting similar in Transcription LLD
FYI @hemantasharma1129

Reply:
Good point. Amended as suggested